PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -0,0 +1,58 @@
+#### Name
+
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
+APP_NAME="Jumpcloud Assist App.app"
+LOCAL_DMG_PATH="/tmp/jumpcloud-assist-app.dmg"
+
+for SIGNAL in TERM KILL; do
+    PID=
+    for _ in $(seq 5); do
+        PID=$(launchctl list | grep com.jumpcloud.assist | cut -f1)
+        if [[ "$SIGNAL" == TERM ]]; then
+            break
+        elif [[ -n "$PID" ]]; then
+            sleep 1
+        fi
+    done
+
+    if [[ -n "$PID" ]]; then
+        kill -s "$SIGNAL" "$PID"
+    else
+        break
+    fi
+done
+
+curl --silent --output "$LOCAL_DMG_PATH" "$REMOTE_DMG_URL" >/dev/null
+VOLUME=$(hdiutil attach "$LOCAL_DMG_PATH" | grep -Eo '(\/Volumes\/.*)')
+
+if ! codesign -dv "$VOLUME/$APP_NAME"; then
+    echo "Application lacks a valid signature, aborting installation"
+    exit 1
+fi
+
+rm -rf "/Applications/$APP_NAME"
+cp -a "$VOLUME/$APP_NAME" /Applications
+hdiutil detach "$VOLUME"
+rm "$LOCAL_DMG_PATH"
+```
+
+#### Description
+
+Downloads and installs the JumpCloud Remote Assist app on a Mac system.
+
+#### *Import This Command*
+
+To import this command into your JumpCloud tenant run the below command using the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Installing-the-JumpCloud-PowerShell-Module)
+
+```
+Import-JCCommand -URL 'TODO'
```

Comment:
https://git.io/ no longer accepting links to shorten :/ I think we want ...

`Import-JCCommand -URL 'https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Remote%20Assist.md`